### PR TITLE
dx-953 included velocitycheck error and common error types in Card other features

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

File: _includes/card-problem-messages.md [hunk 1]
```diff
@@ -5,6 +5,20 @@ problem message. We generally use the problem message `type` and `status` code
...
+
+{:.table .table-striped}
+| Type                           | Status | Description                                 |
+| :----------------------------- | :----: | :------------------------------------------ |
+| `inputerror`                   | `400`  | The server cannot or will not process the request due to an apparent client error (e.g. malformed request syntax, size to large, invalid request).                     |
+| `forbidden`                    | `403`  | The request was valid, but the server is refusing the action. The necessary permissions to access the resource might be lacking.  |
+| `notfound`                     | `404`  | The requested resource could not be found, but may be available in the future. Subsequent requests are permissible.      |
+| `systemerror`                  | `500`  | A generic error message. |
+| `configurationerror`           | `500`  | A error relating to configuration issues.      |
+
```
Comment (NullableInt) [inline _includes/card-problem-messages.md:21]:
```suggestion
| Type                 | Status | Description                                                                                                                                        |
| :------------------- | :----: | :------------------------------------------------------------------------------------------------------------------------------------------------- |
| `inputerror`         | `400`  | The server cannot or will not process the request due to an apparent client error (e.g. malformed request syntax, size to large, invalid request). |
| `forbidden`          | `403`  | The request was valid, but the server is refusing the action. The necessary permissions to access the resource might be lacking.                   |
| `notfound`           | `404`  | The requested resource could not be found, but may be available in the future. Subsequent requests are permissible.                                |
| `systemerror`        | `500`  | A generic error message.                                                                                                                           |
| `configurationerror` | `500`  | A error relating to configuration issues.                                                                                                          |

```

File: _includes/card-problem-messages.md [hunk 2]
```diff
@@ -5,6 +5,20 @@ problem message. We generally use the problem message `type` and `status` code
 to identify the nature of the problem. The problem `name` and `description` will
 often help narrow down the specifics of the problem.
 
+### Common Problem Types
```
Comment (asbjornu) [inline _includes/card-problem-messages.md:8]: These are common across all payment instruments. Perhaps we can create an include for it to reuse everywhere?

File: _includes/card-problem-messages.md [hunk 3]
```diff
@@ -20,6 +34,7 @@ All contract types will have the following URI in front of type:
 | `authenticationstatusrejected` | `403`  | The authentication status was rejected.     |
 | `frauddetected`                | `403`  | The transaction was fraudulent.             |
 | `3dsecuredeclined`             | `403`  | 3-D Secure declined the transaction.        |
+| `velocitycheck`                | `429`  | The limit for how many times a card or different cards can be used for attempting a purchase        |
```
Comment (asbjornu) [inline _includes/card-problem-messages.md:37]:
```suggestion
| `velocitycheck`                | `429`  | Indicates that the limit for how many times a card or different cards can be used for attempting a purchase has been reached.        |
```

Comment (NullableInt) [inline _includes/card-problem-messages.md:37]: Format table:
```| Type                           | Status | Description                                                                                  |
| :----------------------------- | :----: | :------------------------------------------------------------------------------------------- |
| `cardbranddisabled`            | `403`  | The card brand is disabled.                                                                  |
| `accountholdertyperejected`    | `403`  | The account holder type is rejected.                                                         |
| `cardtyperejected`             | `403`  | The card type is rejected.                                                                   |
| `3dsecurerequired`             | `403`  | The transaction was rejected by 3-D Secure.                                                  |
| `authenticationstatusrejected` | `403`  | The authentication status was rejected.                                                      |
| `frauddetected`                | `403`  | The transaction was fraudulent.                                                              |
| `3dsecuredeclined`             | `403`  | 3-D Secure declined the transaction.                                                         |
| `velocitycheck`                | `429`  | The limit for how many times a card or different cards can be used for attempting a purchase |```

**Pull request merged**